react/finger-exercises/redux/02 - react-redux/src/app/screens/App/index.js

```diff
@@ -81,9 +81,7 @@ class App extends Component {
             </div>
           )}
         </div>
-        {this.state.bookSelected.length ? (
-          <ShoppingCart data={this.state.bookSelected} addItem={this.addItem} removeItem={this.removeItem} />
-        ) : null}
+        {this.state.bookSelected.length ? <ShoppingCart /> : null}
```

Comment:
since you no longer use removeItem and addItem, you can remove those declarations from this component